lib/content-services/src/lib/search/components/search-chip-list/directives/search-chip.directive.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Directive, ElementRef, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: '[adf-a11y-search-chip]'
+})
+export class SearchChipDirective {
+    constructor(private el: ElementRef, private renderer: Renderer2) {
+    }
+
+    ngAfterViewInit() {
+        setTimeout(() => {
+            this.renderer.removeAttribute(this.el.nativeElement.parentNode.parentNode.parentNode.parentNode, 'role');
```

Comment:
to avoid this `.parentNode.parentNode.parentNode.parentNode` why not give the directive a target element so one could understand what element is being affected ?